src/Features/CSharp/Portable/Completion/CompletionProviders/AwaitCompletionProvider.cs

```diff
@@ -70,7 +70,7 @@ protected override int GetSpanStart(SyntaxNode declaration)
                 parent = localFunction;
             }
 
-            return parent.Ancestors().FirstOrDefault(node => node.IsAsyncSupportingFunctionSyntax());
+            return parent.AncestorsAndSelf().FirstOrDefault(node => node.IsAsyncSupportingFunctionSyntax());
```

Comment:
VB should be having the same bug.

Reply:
I think this is not an issue in VB. Unlike C#, `SingleLineFunctionLambdaExpression` is not parent of any token. Here's a few tests I believe covered this scenario.
https://github.com/dotnet/roslyn/blob/b3d9ef6c8472794b541641d7e8f30915e43243c9/src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests_AwaitCompletion.vb#L192